### ISSUE TITLE
Roulette Revolver Does not Message_Admins

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#18990]
This is even worse because the explosions it makes don't either if it rolls something that explodes. Very  hard to investigate the cause. It may not be logged either for all I know.

**Comment by Exxion** [comment]
Even the explosions don't message admins? Anyone else thinking runtime?

**Comment by Kurfursten** [comment]
Yes, there were three (3) gun related explosions that shift and none of them sent any admin message. Not even the normal one with the impact size. Other explosions that shift (gibtonite, IED) did get messages.
